lib/slack.ts

```diff
@@ -24,10 +24,21 @@ export const AppMentionSchema = Type.Object({
   ts: Type.String(),
   channel: Type.String(),
   event_ts: Type.String(),
+  thread_ts: Type.Optional(Type.String()),
 });
 export type AppMention = Static<typeof AppMentionSchema>;
 
 export const SlackEventSchema = Type.Object({
   event: Type.Union([ChallengeSchema, AppMentionSchema]),
 });
 export type SlackEvent = Static<typeof SlackEventSchema>;
+
+export async function respondInThread(event: AppMention, text: string) {
```

Comment:
This type can be made more precise so that [other events](https://api.slack.com/events/message) that has a `ts` property can be provided to it too.
```suggestion
interface EventWithTS {
  ts: string;
  thread_ts?: string;
}
export async function respondInThread(event: EventWithTS, text: string) {
```